intercom-java/src/main/java/io/intercom/api/Admin.java

```diff
@@ -30,9 +32,25 @@ public static AdminCollection list()
         return DataResource.list(SENTINEL, "admins", AdminCollection.class);
     }
 
+    public static Admin find(String id)
```

Comment:
Do you think we need a quick test for the find? 
Just wanted to check if you think this is needed or not